docker/start.py

```diff
@@ -9,6 +9,8 @@
 
 import jinja2
 
+VIRTUALENV_INTERPRETER = "/synapse/.venv/bin/python"
```

Comment:
can we just use [`sys.executable`](https://docs.python.org/3/library/sys.html#sys.executable) here rather than hardcoding it?

```suggestion
VIRTUALENV_INTERPRETER = sys.executable
```

(or better yet, inline it below)

Reply:
Done in #12385.